### PR TITLE
Yelp style guide and vote button

### PR DESCRIPTION
**Pull request by gregoryew**
Added Yelp style guide and vote button.  Only the display for vote button so far.  It does not do anything yet.

### CODE REVIEW

File: client/ReviewItem.jsx [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
-    <td>Stars: {props.review.stars}<br />
-    Review: {props.review.text}<br />
-    Useful: {props.review.useful}&nbsp;Funny: {props.review.funny}&nbsp;Cool: {props.review.cool}
-    </td>
-  </tr>
+  <div className="review-wrapper">
+    <div className="review-content">
+      Stars: {props.review.stars}
+      <br />
+      {props.review.text.map(line => <Line line={line} />)}
```
Comment (dylanqian29) [inline client/ReviewItem.jsx:10]: I think you need to add a key when you map through a list. You can use the _id or another way to assign to a key like key = {...} after line ={line}

File: client/ReviewList.jsx [hunk 2]
```diff
@@ -4,14 +4,7 @@ import ReviewItem from './ReviewItem.jsx';
...
     <div>
-      <h4> Reviews </h4>
-    </div>
-    <div>
-      <table>
-        <tbody>
-          {props.reviews.map(review => <ReviewItem review={review} />)}
-        </tbody>
-      </table>
+      {props.reviews.map(review => <ReviewItem review={review} />)}
```
Comment (dylanqian29) [inline client/ReviewList.jsx:7]: same as the comment above, you should use a key for map function

File: db/index.js [hunk 3]
```diff
@@ -11,6 +11,11 @@ const retrieve = (id, callback) => {
         if (err2) {
           callback(err2, null);
         } else {
+          for (let i = 0; i < data.length; i += 1) {
+            let text = data[i].text.split('\n');
+            console.log(text);
```
Comment (dylanqian29) [inline db/index.js:16]: I think you dont need the console.log anymore right?

**Review by dylanqian29 — COMMENTED** [review]
Hey @gregoryew, nice job here. I added some comments with little suggestions. check them out

File: db/index.js (hunk 3 above)
Comment (gregoryew) [reply db/index.js:16]: You are right I will remove. Thanks

File: client/ReviewList.jsx (hunk 2 above)
Comment (gregoryew) [reply client/ReviewList.jsx:7]: I will add a key.  Thanks

File: client/ReviewItem.jsx (hunk 1 above)
Comment (gregoryew) [reply client/ReviewItem.jsx:10]: I will add a key.   Thanks.

**Pull request merged**